homeassistant/components/dynalite/bridge.py

```diff
@@ -61,6 +70,26 @@ def update_device(self, device: Optional[DynaliteBaseDevice] = None) -> None:
         else:
             async_dispatcher_send(self.hass, self.update_signal(device))
 
+    @callback
+    def handle_notification(self, notification: DynaliteNotification) -> None:
+        """Handle a notification from the platform and issue events."""
+        if notification.notification == NOTIFICATION_PACKET:
+            self.hass.bus.fire(
+                f"dynalite_{self.host}_packet",
+                {ATTR_PACKET: notification.data[NOTIFICATION_PACKET]},
+            )
+        if notification.notification == NOTIFICATION_PRESET:
+            self.hass.bus.fire(
```

Comment:
We're in async context so need to use the async api.
```suggestion
            self.hass.bus.async_fire(
```

Reply:
thanks